evcxr/src/command_context.rs

```diff
@@ -159,6 +166,55 @@ Panic detected. Here's some useful information if you're filing a bug report.
         }
     }
 
+    fn execute_shell_command(
+        &mut self,
+        shell_command: &ShellCommand,
+    ) -> Result<EvalOutputs, Error> {
+        use std::process::Command;
+
+        let command_output = Command::new("sh")
+            .arg("-c")
+            .arg(&shell_command.command)
+            .output();
+
+        match command_output {
+            Ok(output) => {
+                let stdout_str = String::from_utf8_lossy(&output.stdout);
+                let stderr_str = String::from_utf8_lossy(&output.stderr);
+
+                let mut eval_outputs = EvalOutputs::default();
+
+                // Add the stdout and stderr to eval_outputs
+                eval_outputs
+                    .content_by_mime_type
+                    .insert("stdout".to_string(), stdout_str.into());
```

Comment:
The keys in this map are mime types - i.e. "text/plain" rather than "stdout". Perhaps concatenate stdout and stderr into the one output. e.g.

```rust
eval_outputs.content_by_mime_type.insert(
                    "text/plain".to_string(),
                    stdout_str.into_owned() + stderr_str.as_ref(),
                );
```

The way it was, I didn't see any output when testing, but once changed to emit as "text/plain", the output showed up. I tested from the REPL and using jupyter notebook from a browser. I'm assuming it worked for you as it was - perhaps you were testing from a different jupyter frontend that maybe displays content with unknown mime types.

Reply:
Rather than concatenating stdout and stderr, I think a single pipe or file should be used for both stdout and stderr to preserve the ordering of both just like it would in a terminal.